File: wEBcMD/ClientApp/src/impl/FindAdresses.ts
import { Guid} from "guid-typescript";
import { AdressDTO } from "../api/AdressDTO";
import { FindAdressesBase } from "../api/FindAdressesBase";
import { CommandDTO } from "../api/CommandDTO";

/**
 * Addresses search, with multiple tokens
 */
export class FindAdresses extends FindAdressesBase {

   constructor(dto?: CommandDTO, type?: Guid){super(dto, type ? type : FindAdressesBase.TypeId)}

};

File: wEBcMD/Impl/FindAdresses.cs
using System;
using System.Reflection;
using System.Collections.Generic;

namespace wEBcMD
{
   public partial class FindAdressesWrapper : CommandWrapper
   {
      /// <summary>
      /// Addresses search, with multiple tokens
      /// </summary>

		public partial List<AdressDTO> FindAdresses(String searchText)
		{
			Log.Trace($"Implementation in {MethodBase.GetCurrentMethod()}");
			return default;
		}
   };

}
